workspaces/redhat-resource-optimization/plugins/redhat-resource-optimization/src/pages/openshift/components/Filters.tsx

```diff
@@ -351,7 +354,11 @@ export function Filters(props: FiltersProps) {
         {/* Filter table by */}
         <div className={classes.filterSection}>
           <div
-            style={{ backgroundColor: '#F2F2F2', padding: 16, borderRadius: 8 }}
+            style={{
+              backgroundColor: isDarkMode ? '#262626' : '#F2F2F2',
```

Comment:
@asmasarw do you think we can make use of https://mui.com/material-ui/customization/palette/ for such changes?